### ISSUE TITLE
Decouple the follow-up visits from the follow-up period, or allow the creation of follow-up visits at any time

### ISSUE THREAD

**Issue by ThomasVitus** [SORMAS-Foundation/SORMAS-Project#6575]
### Feature Description
To be able to create a follow-up visit after the end of the follow-up period, the follow-up period must first be extended, otherwise saving a call is not possible.

### Problem Description
We also document e.g. telephone calls regarding the issuance of certificates for a long time after the successful completion of the quarantine. It is then always inconvenient not to be able to save the information on the call and to have to extend the follow-up period first. Maybe I am missing an important context here?

![Screenshot_2021-09-01 SORMAS](https://user-images.githubusercontent.com/86602055/131660733-fd49c1bb-5c47-45d7-be28-d6b344f67dd0.png)

![Screenshot_2021-09-01 SORMAS(1)](https://user-images.githubusercontent.com/86602055/131660739-204267f2-21a5-4483-9259-76a06d5a275f.png)

### Proposed Change
Please decouple the follow-up visits from the follow-up period, or allow the creation of follow-up visits at any time

**Comment by MateStrysewske** [comment]
@ThomasVitus This is definitely something that needs to be discussed. The validation here makes sure that the visits are properly associated with the contacts in the first place, the reason being that there is no hard link between contacts and visits as one person can have multiple contacts with the same disease at the same time, and the visits need to show up in all of them. Because of that, if you would create a visit for a contact months after the contact took place, that visit would not show up in the visit list of that contact.

**Comment by ThomasVitus** [comment]
@MateStrysewske Yes, that's exactly the point, that in my proposal the epidemiological context is abandoned and the follow-up visits are used instead in an administrative context. And possibly months after the quarantine. But since SORMAS is a tool for epidemic control and not one of certificates issued afterwards, I would withdraw my proposal :)
